pkg/cache/context.go

```diff
@@ -503,6 +504,20 @@ func (ctx *Context) triggerReloadConfig() {
 	}
 }
 
+func (ctx *Context) UpdatePodCondition(pod *v1.Pod, condition *v1.PodCondition) error {
+	log.Logger.Info("Updating pod condition",
+		zap.String("namespace", pod.Namespace),
+		zap.String("name", pod.Name),
+		zap.String("", string(condition.Type)),
+		zap.String("status", string(condition.Status)),
+		zap.String("reason", condition.Reason))
```

Comment:
Should we not just log the whole condition here?
If not we need a description for the `Type` log

Reply:
Changed it to zap.Any("podCondition", condition)